docs/testing-guidelines/getting-code-coverage.md

```diff
@@ -0,0 +1,127 @@
+Getting Code Coverage Analysis for PowerShell
+===
```

Comment:
I believe we've standardized on using hashes for headers

Reply:
I'll change it over, although my opinion is that the hashes are less readable in the raw view.